order/supplier/receive/simple-order-event.md

```diff
@@ -0,0 +1,249 @@
+---
+description: How to receive a simple order sent by the buyer.
+---
+
+# Receive an order
+
+Tradecloud will send a purchase order, either new or updated, to the supplier when an order event has been triggered.
+
+## `simpleOrderEvent`
+
+This page assumes you are using the webhook with the simple delivery schedule in the `simpleOrderEvent`.
```

Comment:
I'd move this up for clarification.
Also; what if this is not the case?

```suggestion
This page assumes the supplier integrated by using the webhook, using the simple delivery schedule in the `simpleOrderEvent`.
```

Reply:
Moved up and added a back link to the README